.travis.yml

```
@@ -88,7 +84,9 @@ env:
 
 install:
   - pip install "pip>=7.1.0"
-  - pip install tox-travis tox codecov coverage
+  - pip install tox-travis tox codecov
+  # See https://github.com/nedbat/coveragepy/issues/915
+  - pip install "coverage<5.0"
```

IMO this one can be reverted now, a fixed coverage 5.1 is out.
```suggestion
  - pip install tox-travis tox codecov coverage
```